Minio/Credentials/AssumeRoleProvider.cs

```diff
@@ -159,7 +159,7 @@ internal override async Task<HttpRequestMessageBuilder> BuildRequest()
         });
         var byteArrContent = await formContent.ReadAsByteArrayAsync();
         requestMessageBuilder.SetBody(byteArrContent);
-        requestMessageBuilder.AddOrUpdateHeaderParameter("Content-Type", "application/x-www-form-urlencoded");
+        requestMessageBuilder.AddOrUpdateHeaderParameter("Content-Type", "application/x-www-form-urlencoded; charset=utf-8");
```

Comment:
The following change is required for regitlint check. Otherwise, Linux build will choke on it.

Either do the following change manually and push, or run regitlint yourself and push.
```suggestion
        requestMessageBuilder.AddOrUpdateHeaderParameter("Content-Type",
            "application/x-www-form-urlencoded; charset=utf-8");
```

This will fix the Linux build failure.